core/arch/arm/kernel/thread_spmc.c

```diff
@@ -1843,27 +1848,51 @@ void thread_spmc_relinquish(uint64_t cookie)
 		EMSG("Failed to relinquish cookie %#"PRIx64, cookie);
 }
 
-static int set_pages(struct ffa_address_range *regions,
-		     unsigned int num_regions, unsigned int num_pages,
+static int add_pages_at(struct ffa_address_range *regions,
+		     unsigned int num_regions, unsigned int *idx,
```

Comment:
indenation issue here also (sorry, i guess i missed it)

Reply:
Done